src/app/[orgSlug]/study/[studyId]/view/job-results-status-message.tsx

```diff
@@ -58,6 +61,16 @@ export const JobResultsStatusMessage: FC<JobResultsStatusMessageProps> = ({ job,
     return (
         <Stack>
             <Text>{message}</Text>
+            {errorMessage && (
```

Comment:
I do not think we want to show the errorMessage to researchers.  Instead we should show it to reviewers in the [studyId]/review/study-results.tsx component